maintainers/datasets.md

```diff
@@ -50,13 +50,13 @@ Finally, if neither the build option nor the environment variable are set, tests
 ## Writing Tests That Use Datasets
 
 Test developers can use the RAPIDS dataset locating utilities by calling the following function:
-```
+```cpp
 const std::string& get_rapids_dataset_root_dir()
 ```
 `get_rapids_dataset_root_dir()` will return a `std::string` reference to the root directory containing the datasets, as determined by the user-accessible options described above, or a default value of `/datasets` if the user did not use the options.
 
 ### Example API usage
-```
+```cpp
```

Comment:
I worry this leads to a bit low contrast for these code boxes. Contrast ratio is ~2.5 and recommended is 4.5+. https://webaim.org/resources/contrastchecker/

No idea how customizable these code formats are though (I assume non-customizable.) 

![image](https://github.com/rapidsai/docs/assets/104460670/31914851-551f-4306-9ac4-0da8e8b19f31)
![image](https://github.com/rapidsai/docs/assets/104460670/4b92a75b-9d28-4b6b-83f5-2412f35d9018)

Reply:
Updated! Try again:
https://deploy-preview-494--docs-rapids-ai.netlify.app/maintainers/datasets/#writing-tests-that-use-datasets
https://deploy-preview-494--docs-rapids-ai.netlify.app/install#wsl2-docker

### Names and Parentheses
![image](https://github.com/rapidsai/docs/assets/3943761/bccf8be9-6002-408c-8214-9ba16fb91e20)

### Operators
![image](https://github.com/rapidsai/docs/assets/3943761/3035076b-48ee-44ab-9c68-8d27949c6d96)

Reply:
Love it - thank you!